decisions/DR-0001-architecture-clarifications.md

```diff
@@ -0,0 +1,63 @@
+# Architecture clarifications
+
+| Status     | draft                                                             |
+|:-----------|:------------------------------------------------------------------|
+| **PR #**   | [748](https://github.com/blindnet-io/product-management/pull/748) |
+| **Author** | Amaury Rousseau (amaury@blindnet.io)                              |
+
+## Context and Problem Statement
+
+The [High Level Conceptualization](https://github.com/blindnet-io/product-management/blob/d3dad802ce52fb1488cbda8302d3c36c94c3b47b/refs/high-level-conceptualization/README.md),
+[High Level Architecture](https://github.com/blindnet-io/product-management/blob/d3dad802ce52fb1488cbda8302d3c36c94c3b47b/refs/high-level-architecture/readme.md),
+and the [Architecture specification](https://github.com/blindnet-io/product-management/blob/ae344653217b03d3857fba63f5d716e43c72263b/specifications/architecture/README.md)
+leave some free space for interpretation when implementing the devkit components.
+To enable an efficient, coordinated and coherent development of all components, some choices have to be made.
+
+## Considered Options
+
+This record is the result of multiple meetings to discuss this matter between @filipblnt, @m4rk055 and @TheKinrar.
+
+## Decision Outcome
+
+### Interaction of Data Capture and Storage components
+
+The Storage component only acts as an abstract interface to any backing storage implementation.
+It is able to store data as byte streams, which might be short text representations, like Data Capture metadata,
```

Comment:
```suggestion
It allows to link actual data records with PRIV metadata corresponding to it. It also allows to interpret actions resulting from Privacy Computation Engine, intro instructions (e.g. SQL queries) executable over data stroage.
```
After our discussion today, I think we are limiting the role of Storage to only an interface to link with other part of the architecture, so I suggest this change